src/App.vue

```diff
@@ -619,6 +661,19 @@ export default {
 				this.$router.push({ name: 'root' })
 			}
 		},
+
+		/**
+		 * Emits the UNREAD_COUNT_UPDATED event with the current counter values
+		 */
+		emitUnreadCountUpdated() {
+			const eventData = {
+				totalUnreadMessages: this.getTotalUnreadMessages,
+				totalUnreadMentions: this.getTotalUnreadMentions,
+				totalUnreadMentionsDirect: this.getTotalUnreadMentionsDirect
+			}
+
+			emit(EVENTS.UNREAD_COUNT_UPDATED, eventData)
```

Comment:
To simplify naming a bit and remove the constant (see comment in the `talk-desktop`)
- The entire event payload is about counters, the total number of unread _something_, so we can omit it from every property name
- Let's add `conversations` in case a user would like to see the number of conversations, but not total messages in all the conversations
- Event is renamed to `unreads` to be plural (it's "unreads" that were updated)

```suggestion
			emit('talk:unreads:updated', {
				conversations: /* Number of conversations with unread messages */,
				messages: this.getTotalUnreadMessages,
				mentions: this.getTotalUnreadMentions,
				mentionsDirect: this.getTotalUnreadMentionsDirect
			})
```